docs/README.md

```diff
@@ -99,3 +99,55 @@ $ git commit
 Create a pull request once you are ready to submit your changes.
 
 We will review your changes, add any feedback and once everything looks good merge your changes into the mainline.
+
+
+## Advanced Usage
+
+### Re-generating diagrams for API docs
+
+Generate new grammars and diagrams when updating the docs as follows:
+
+_Note: Modifications will typically be in the `docs/content/latest/api/` either `ysql` or `ycql` subfolder.
+Will assume that the subfolder is `docs/content/latest/api/ysql` and the target file is `commands/cmd_copy.md` in the examples below_
+
+1. Update (edit) the appropriate `ebnf` source grammar file with your changes (e.g. adding a new supported statement or clause).
```

Comment:
1. Update the appropriate EBNF source grammar file (`<source>_grammar.ebnf`) with your changes, for example, adding support for a new statement or clause.